### ISSUE TITLE
rename env variables in the pipenv recipe

### ISSUE THREAD

**Issue by scott-vsi** [VisionSystemsInc/docker_recipes#35]
We need to rename the PIPENV_VERSION environment variable used by the [30_get-pipenv script](https://github.com/VisionSystemsInc/docker_recipes/blob/abf84c22ed896c05821938dde44286fddfbbe0b6/30_get-pipenv#L174) because it conflicts with a variable of the same name used by PIPENV.

It looks like [2018.11.14](https://github.com/pypa/pipenv/blob/master/CHANGELOG.rst#features--improvements-4)  "added persistent settings for all CLI flags via PIPENV_{FLAG_NAME} environment variables". If we export PIPENV_VERSION and try to run a pipenv command, we get this error:

```
Error: Invalid value for '--version': 2020.11.15 is not a valid boolean
```

**Comment by scott-vsi** [comment]
As a workaround, export this env variable here until this issue is fixed: 

```
RUN export PIPENV_VERSION=2020.11.15; export VIRTUALENV_VERSION=20.3.1; \
    for patch in /usr/local/share/just/container_build_patch/*; do "${patch}"; done
```

**Comment by drewgilliam** [comment]
fixed by PR #40
